docs/topics/Gateway.md

```diff
@@ -46,25 +46,25 @@ Packets sent from the client to the Gateway API are encapsulated within a [gatew
 
 ### Receiving Payloads
 
-Receiving payloads with the Gateway API is slightly more complex than sending. When using the JSON encoding with compression enabled, the Gateway has the option of sending payloads as compressed JSON binaries using zlib, meaning your library _must_ detect (see [RFC1950 2.2](https://tools.ietf.org/html/rfc1950#section-2.2)) and decompress these payloads before attempting to parse them. The gateway does not implement a shared compression context between messages sent.
+Receiving payloads with the Gateway API is slightly more complex than sending. When using the JSON encoding with compression enabled, the Gateway has the option of sending payloads as compressed JSON binaries using Zlib, meaning your library _must_ detect (see [RFC1950 2.2](https://tools.ietf.org/html/rfc1950#section-2.2)) and decompress these payloads before attempting to parse them. The Gateway does not implement a shared compression context between messages sent.
```

Comment:
capitalization is not needed here

Reply:
Why is the `gateway` in the first sentence capitalized then? But I agree, my capitalization seems wrong?